<?xml version="1.0"?>
<launch>

  <param name="robot_description"
         command="xacro $(find sick_safetyscanners)/model/urdf/example.urdf.xacro">
  </param>

  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
  <node name="joint_state_publisher" type="joint_state_publisher" pkg="joint_state_publisher"/>

  <node  pkg="rviz" type="rviz" output="screen" name="rviz"/>

  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="world_name" value="worlds/empty.world"/>
    <!-- Always start in paused mode, and only unpause when spawning the model -->
    <arg name="paused" value="false"/>
    <arg name="gui" value="true"/>
    <arg name="use_sim_time" value="true"/>
  </include>

  <node name="laser_scanners_model_spawner"
        pkg="gazebo_ros"
        type="spawn_model"
        args="-param robot_description -urdf -model scanners"/>

	<!-- <node pkg="ira_laser_tools" name="laserscan_multi_merger" type="laserscan_multi_merger" output="screen">
    <param name="destination_frame" value="world"/>
    <param name="cloud_destination_topic" value="/merged_cloud"/>
    <param name="scan_destination_topic" value="/scan_multi"/>
    <param name="laserscan_topics" value ="/sick_safetyscanners_front/scan /sick_safetyscanners_rear/scan" /> 
    <param name="angle_min" value="0"/>
    <param name="angle_max" value="3.1457"/>
    <param name="angle_increment" value="0.008746"/>
    <param name="scan_time" value="0.0"/>
    <param name="range_min" value="0.5"/>
    <param name="range_max" value="20.0"/>
	</node> -->

</launch>